mux_command.go

```diff
@@ -167,9 +170,39 @@ func (m *CommandMux) HandleEvent(b *Bot, msg *irc.Message) {
 		newEvent.Command = newEvent.Command[len(m.prefix):]
 	}
 
+	var commands []string
+	var distances []int
 	if b.FromChannel(newEvent) {
 		m.public.HandleEvent(b, newEvent)
+		commands = m.public.eventNames()
 	} else {
 		m.private.HandleEvent(b, newEvent)
+		commands = m.public.eventNames()
+	}
+
+	if m.displaySimilar {
+		for _, v := range commands {
+			distances = append(distances, levenshtein.ComputeDistance(newEvent.Command, v))
+		}
+
+		minDistance := minIntSlice(distances)
+
+		// 0 means it matched exactly so we don't need recommendations
+		if minDistance == 0 {
+			return
+		}
+
+		var similarCommands []string
+		for k, v := range distances {
+			if v-minDistance < 1 {
```

Comment:
Worth making this configurable?

Reply:
The responses are pretty terrible when you get much bigger than 1... might be worth it though

Reply:
Ah yeah, very fair point